### ISSUE TITLE
Cookie consent buttons cropped at 200% text scale

### ISSUE THREAD

**Issue by ravenokko** [shopware/shopware#16056]
### Shopware Version

trunk

### Affected area / extension

Platform(Default)

### Actual behaviour

Text for `Only technically required` and `Close cookie preferences` buttons are cut off at `320px` width with text zoomed to 200%. This does not meet the requirements for [WCAG S.C. 1.4.4](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html).

<img width="321" height="600" alt="Image" src="https://github.com/user-attachments/assets/547542e4-2b5c-4bc4-b118-b9c3bdb3bac8" />
<img width="321" height="728" alt="Image" src="https://github.com/user-attachments/assets/8a70666c-c297-495e-a86c-ee0993492800" />

### Expected behaviour

Button text wraps for appropriately. The `X` icon beside `Close cookie preferences` should also remain visible.

### How to reproduce

- In Firefox go to Preferences > General > Zoom
  - Ensure default Zoom is set to 100%
  - check `Zoom text only`
- Set viewport to `320px` wide
- Zoom to 200% on storefront homepage
